test/api/unit/models/site.test.js

```diff
@@ -1,102 +1,125 @@
-const expect = require("chai").expect
-const sinon = require("sinon")
-const config = require("../../../../config")
-const factory = require("../../support/factory")
-const { Site } = require("../../../../api/models")
+const expect = require('chai').expect;
```

Comment:
The many changes to single/double quotes prevent easy detections of the changes to this file. Could you describe them in a sentence or two, or point to them in comments?